### PR TITLE
improved key function usage

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: service/key/key.go [hunk 1]
```diff
@@ -20,18 +20,10 @@ func ClusterCustomer(customObject kvmtpr.CustomObject) string {
 	return customObject.Spec.Cluster.Customer.ID
 }
 
-func ClusterDomain(sub, clusterID, domain string) string {
-	return fmt.Sprintf("%s.%s.%s", sub, clusterID, domain)
-}
-
```
Comment (xh3b4sd) [inline service/key/key.go]: These functions are not used at all so I removed them.

File: service/resource/deployment/master_deployment.go [hunk 2]
```diff
@@ -126,7 +126,7 @@ func newMasterDeployments(customObject kvmtpr.CustomObject) ([]*extensionsv1.Dep
 								Command: []string{
 									"/opt/k8s-endpoint-updater",
 									"update",
-									"--provider.bridge.name=" + key.NetworkBridgeName(key.ClusterID(customObject)),
+									"--provider.bridge.name=" + key.NetworkBridgeName(customObject),
```
Comment (xh3b4sd) [inline service/resource/deployment/master_deployment.go]: The usage of the key functions is now a bit easier.

File: service/key/key.go (hunk 1 above)
Comment (MarcelMue) [reply service/key/key.go]: I am a bit confused, why do these functions exist if they aren't used? Dead weight from previous changes?

File: service/resource/deployment/master_deployment.go (hunk 2 above)
Comment (MarcelMue) [reply service/resource/deployment/master_deployment.go]: I like this 👍

File: service/key/key.go (hunk 1 above)
Comment (xh3b4sd) [reply service/key/key.go]: I think somebody forget to remove them when the used code got changed. I don't know.

**Pull request merged**